Resources/Storage/EchoStorage.py

```diff
@@ -0,0 +1,118 @@
+""" :mod: EchoStorage
```

Comment:
sphinx autodoc will take care of creating the documentation strings for modules, classes etc. So to not cause warnings, please just put
```
"""Echo module based on the GFAL2_StorageBase class."""
```